src/daemon/bitcoind/actions.rs

```diff
@@ -291,11 +291,10 @@ fn comprehensive_rescan(
                     MIN_CONF,
                 );
                 db_unconfirm_deposit_dbtx(db_tx, vault.id)?;
-                assume_some!(
-                    deposits_cache.get_mut(&vault.deposit_outpoint),
-                    "Db vault not in cache?"
-                )
-                .status = VaultStatus::Unconfirmed;
+                deposits_cache
```

Comment:
:%s/assume_ok/assume_some in the commit message

Reply:
Yeah... I meant to clean both but you are right i only pruned `_some`s! Done.

Reply:
Hope CI roulette not gonna bite me now :sob: